contracts/test/_fixture.js

```diff
@@ -815,6 +856,34 @@ async function morphoAaveFixture() {
   return fixture;
 }
 
+/**
+ * Configure a Vault with only the Morpho strategy.
+ */
+function oethMorphoAaveFixtureSetup() {
+  return deployments.createFixture(async () => {
+    const fixture = await oethDefaultFixture();
+
+    if (isFork) {
+      const sGuardian = await ethers.provider.getSigner(
+        addresses.mainnet.Guardian
+      );
+
+      await fixture.oethVault
+        .connect(sGuardian)
+        .setAssetDefaultStrategy(
```

Comment:
note: if Transfer Governance Pr https://github.com/OriginProtocol/origin-dollar/pull/1541 gets merged in first `sGuardian` will need to be changed to `sGovernor` - old mainnet timelock contract.